tests/test_calculator.hh

```diff
@@ -423,8 +423,8 @@ namespace rascal {
         "reference_data/small_molecule.json",
         "reference_data/methane.json"};
     // Simpler structures for debugging:
-    //"reference_data/diamond_2atom.json",
-    //"reference_data/SiC_moissanite.json",
+    // "reference_data/diamond_2atom.json",
+    // "reference_data/SiC_moissanite.json",
```

Comment:
Ugh, was this `cpplint` again? This extra space can make uncommenting a real PITA.

Reply:
yes, cpplint.

You can configure your editor to comment / uncomment with spaces automatically.

Reply:
I know, but it’s still a pain. I’m used to thinking of comments with a space as “text comments” (explanatory text in plain English) and those without a space as temporarily disabled code.